avocado/core/runner.py

```diff
@@ -512,7 +512,8 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
         test_result_total = variants.get_number_of_tests(test_suite)
         no_digits = len(str(test_result_total))
         self.result.tests_total = test_result_total
-        self.result.start_tests()
+        self.job._result_events_dispatcher.map_method('start_tests',
+                                                      self.result)
```

Comment:
Also you need to add this `start_tests` into the `avocado.core.remote.runner`, right?